axoupdater/src/test/helpers.rs

```diff
@@ -0,0 +1,114 @@
+use std::{
+    env::consts::EXE_SUFFIX,
+    path::PathBuf,
+    process::{Command, Stdio},
+};
+
+static RECEIPT_TEMPLATE: &str = r#"{"binaries":[BINARIES],"install_prefix":"INSTALL_PREFIX","provider":{"source":"cargo-dist","version":"0.10.0-prerelease.1"},"source":{"app_name":"APP_NAME","name":"PACKAGE","owner":"axodotdev","release_type":"github"},"version":"VERSION"}"#;
```

Comment:
release_type: "github"

*needs* to be configurable. It would be great if cargo-dist had a test for each path, since it works with both!

Reply:
I realized the "owner" is also hardcoded, so I'll un-hardcode that too.